docs/application-setup.md

```diff
@@ -1,49 +1,37 @@
-# Application setup
+# Luigi Core Installation
 
-Prior to start developing with Luigi, you need to set up your application. This document shows you how to set up a web application using the Luigi micro frontend framework.
+This document shows you how to set up a web application using the Luigi micro frontend framework. This involves several basic steps:
+* Adding Luigi's `npm` packages to your project dependencies
+* Giving Luigi exclusive control over your entry `index.html` file
+* Starting the server to run your application
 
-Choose the framework to build your application:
-
-[Application setup without a framework](#noframework)  
-[Angular 6](#angular6)  
-[SAPUI5/OpenUI5](#sapui5)  
-[VUE.JS](#vuejs)
-
-## Basic application setup
+All those steps are condensed in a single installer script which sets up a Luigi application on your system. Keep in mind that if you have a single page application, you need to transfer it to a two-page application. Your application should run to make sure the existing router doesn't interfere.
```

Comment:
The last sentence is a bit confusing to me: does it mean that the application needs to be running to check if there is no interference with the router or that you should run it to see if there is no such interference?

Reply:
It's the second one :) I changed it to "You should run your application to make sure the existing router doesn't interfere."

Reply:
Then I would put it this way - it's more direct and clear.